sys-apps/systemd/files/256-bpf-gcc.patch

```diff
@@ -0,0 +1,10 @@
+--- a/meson.build
++++ b/meson.build
+@@ -1109,6 +1109,7 @@ else
+         elif bpf_compiler == 'gcc'
+                 bpf_gcc = find_program('bpf-gcc',
+                                        'bpf-none-gcc',
++                                       'bpf-unknown-none-gcc',
+                                        required : true,
+                                        version : '>= 13.1.0')
+                 bpf_gcc_found = bpf_gcc.found()
```

Comment:
is this not redundant with the -D that's being passed?

Reply:
No, it's still needed, as we just say "use GCC", and then it tries to find a GCC, rather than it taking the name of a binary to use.

Reply:
oh, duh - OK